plrust/src/target.rs

```diff
@@ -104,27 +113,83 @@ impl CompilationTarget {
     }
 }
 
-pub(crate) fn tuple() -> Result<&'static CompilationTarget, &'static TargetErr> {
-    pub(crate) static TARGET_TUPLE: Lazy<Result<CompilationTarget, TargetErr>> =
-        Lazy::new(|| match env::var("PLRUST_TARGET") {
-            Ok(v) => Ok(v.into()),
-            Err(env::VarError::NotPresent) => {
-                cfg_if::cfg_if! {
-                    if #[cfg(all(feature = "target_postgrestd",
-                        any(target_arch = "x86_64", target_arch = "aarch64"),
-                        target_os = "linux",
-                        target_env = "gnu"))]
-                    {
-                        Ok(host::target_tuple().into())
-                    } else if #[cfg(feature = "target_postgrestd")] {
-                        Err(TargetErr::Unsupported)
-                    } else {
-                        Ok(host::target_tuple().into())
-                    }
-                }
-            }
-            Err(env::VarError::NotUnicode(s)) => Err(TargetErr::InvalidSpec(s)),
+#[derive(Clone, Copy, PartialEq, Hash, Debug)]
+pub(crate) enum CrossCompilationTarget {
+    X86_64 = 1,
+    Aarch64 = 2,
+}
```

Comment:
Why are we assigning values to these variants? We usually do that in enums we want to be FFI compatible, but there is no repr attribute here.

Reply:
Oversight from an earlier incarnation. Will fix. Good catch.

Reply:
Yeah, I'm not so concerned about doing so as much as I was just checking that it wasn't being used in an odd way elsewhere that escaped my notice.